buckw

```diff
@@ -213,10 +213,15 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    for arg in "$@"
```

Comment:
I mostly did it this way for any future argument checking. I'll go ahead and do it your simplified way.